package.json

```diff
@@ -17,13 +17,15 @@
     "eslint-config-taskcluster": "^3.0.0",
     "taskcluster-client": "^12.0.0",
     "taskcluster-lib-docs": "^10.0.1",
+    "taskcluster-lib-iterate": "^11.0.0",
     "taskcluster-lib-loader": "^11.0.0",
     "taskcluster-lib-monitor": "^11.1.1",
+    "taskcluster-lib-urls": "^12.0.0",
```

Comment:
I don't think you need this... We already have this in the root `package.json` (in fact, all the dependencies whose name start with `taskcluster-lib` can be safely removed from here, I believe. Same goes for `taskcluster-client` and `typed-env-config`). ALso, we don't use `eslint-config-taskcluster` anymore, so also can go.

@imbstack am I correct?